src/utils/platform.ts

```diff
@@ -0,0 +1,11 @@
+import UAParser from 'ua-parser-js';
+
+export const getPlatform = () => {
+  const ua = navigator.userAgent;
```

Comment:
this seems break on server mode, because navigator is undefined on node.

Reply:
try it, it seems work correctly...

Reply:
I will resolve it tomorrow.